cfgov/jinja2/v1/_queries/filtered_17.json

```diff
@@ -0,0 +1,14 @@
+{
+  "filtered": {
+    "filter": {
+      "or": [
+      ]
+    },
+    "query": {
+      "bool": {
+        "should": [
+        ]
+      }
+    }
+  }
+}
```

Comment:
using ES 1.7 format. Will need to be edited if we ever make an ES upgrade to 2.2